webapp/home/labs/genome/annotation.yml

```diff
@@ -135,6 +135,136 @@ tabs:
         view_link: ''
         view_tip: ''
         button_tip: Run in Galaxy AU
+        
+      subsections:
```

Comment:
The `subsections` key must replace the entire content of `content` for this to work. You can't declare `subsections` in the middle of the content. To fix this, the preceding items should be put into their own subsection (e.g. "General"). Like:

```yaml
  - id: workflows
    title: Workflows
    heading_html: >
      A workflow is a series of Galaxy tools that have been linked together to perform a specific analysis. You can use and customize the example workflows below.
      <a href="https://galaxyproject.org/learn/advanced-workflow/" target="_blank">Learn more.</a>
    content:
      subsections:
        - id: general
          title: General use
          content:
            - title_html: Annotation with Maker
```